packages/cli/src/commands/setup/chakra-ui/chakra-ui.js

```diff
@@ -0,0 +1,100 @@
+import fs from 'fs'
+import path from 'path'
+
+import execa from 'execa'
+import Listr from 'listr'
+
+import { getPaths } from '../../../lib'
+import c from '../../../lib/colors'
+
+import { checkSetupStatus, wrapWithChakraProvider } from './tasks/setup-chakra'
+
+export const command = 'chakra-ui'
+export const description = 'Setup Chakra UI'
+
+export function builder(yargs) {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+  yargs.option('install', {
+    alias: 'i',
+    default: true,
+    description: 'Install packages',
+    type: 'boolean',
+  })
+}
+
+export async function handler({ force, install }) {
+  const packages = [
+    '@chakra-ui/react',
+    '@emotion/react@^11',
+    '@emotion/styled@^11',
+    'framer-motion@^4',
```

Comment:
We had some issue with breaking changes of our peer dependencies. 
To prevent a bad DX when starting with chakra-ui we suggest to install the peer dependencies with a pinned major version.